docs/platforms/javascript/common/options.mdx

```diff
@@ -0,0 +1,454 @@
+
+</SdkOption>
+
+<SdkOption name="attachStacktrace" type='boolean' defaultValue='false'>
+
+When enabled, stack traces are automatically attached to all messages logged. Stack traces are always attached to exceptions; however, when this option is set, stack traces are also sent with messages. This option, for instance, means that stack traces appear next to all log messages.
+
+Grouping in Sentry is different for events with stack traces and without. As a result, you will get new groups as you enable or disable this flag for certain events.
+
+</SdkOption>
+
+<SdkOption name="serverName" type='string' categorySupported={['server', 'serverless']}>
+
+This option can be used to supply a "server name." When provided, the name of the server is sent along and persisted in the event. For many integrations, the server name actually corresponds to the device hostname, even in situations where the machine is not actually a server.
+
+Most SDKs will attempt to auto-discover this value.
+
+</SdkOption>
+
+<SdkOption name="ignoreErrors" type='Array<string | RegExp>' defaultValue='[]'>
+
+<Include name="platforms/configuration/options/ignore-errors.mdx" />
+
+</SdkOption>
+
+<SdkOption name="ignoreTransactions" type='Array<string | RegExp>' defaultValue='[]'>
+
+<Include name="platforms/configuration/options/ignore-transactions.mdx" />
+
+</SdkOption>
+
+<SdkOption name="denyUrls" type='Array<string | RegExp>' defaultValue='[]' categorySupported={['browser']}>
+
+<Include name="platforms/configuration/options/deny-urls.mdx" />
+
+</SdkOption>
+
+<SdkOption name="allowUrls" type='Array<string | RegExp>' defaultValue='[]'  categorySupported={['browser']}>
+
+<Include name="platforms/configuration/options/allow-urls.mdx" />
+
+</SdkOption>
+
+<PlatformSection supported={['javascript.electron']}>
+<SdkOption name="autoSessionTracking" type='boolean' defaultValue='true'>
+
+When not set to `false`, the SDK tracks sessions linked to the lifetime of the Electron main process.
+
+</SdkOption>
+</PlatformSection>
+
+<SdkOption name="initialScope" type="CaptureContext">
+
+Data to be set to the initial scope. Initial scope can be defined either as an object or a callback function, as shown below.
+
+```javascript
+// Using an object
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  initialScope: {
+    tags: { "my-tag": "my value" },
+    user: { id: 42, email: "john.doe@example.com" },
+  },
+});
+
+// Using a callback function
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  initialScope: (scope) => {
+    scope.setTags({ a: "b" });
+    return scope;
+  },
+});
+```
+
+</SdkOption>
+
+<SdkOption name="maxValueLength" type='number' defaultValue='250'>
+
+Maximum number of characters a single value can have before it will be truncated.
```

Comment:
l: probably a bit clearer? (feel free to disregard)

```suggestion
Maximum number of characters a every string property on events sent to Sentry can have before it will be truncated.
```